picard/cluster.py

```diff
@@ -94,6 +100,10 @@ def remove_file(self, file):
         self.item.remove_file(file)
         if not self.special and self.get_num_files() == 0:
             self.tagger.remove_cluster(self)
+        self.update_metadata_images()
+        if self.related_album:
```

Comment:
`add_file()` and `add_files()` share a lot of code, we can reduce code redundancy, but perhaps after merging this PR (since it isn't its purpose).